### PR TITLE
New strategy: voting power for NFT xSGT [xsgt-voting-power]

### PR DESCRIPTION
**Pull request by acinmavi**
Changes proposed in this pull request:
- Add new strategy: voting power for NFT xSGT

### CODE REVIEW

File: src/strategies/xsgt-voting-power/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bluebelt';
```
Comment (ChaituVR) [inline src/strategies/xsgt-voting-power/index.ts:5]:
```suggestion
export const author = 'acinmavi';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@acinmavi `contract-call` strategy should work perfectly here, Have you tried it? let me know if it doesn't work :)

File: src/strategies/xsgt-voting-power/index.ts (hunk 1 above)
Comment (acinmavi) [reply src/strategies/xsgt-voting-power/index.ts]: Thanks @ChaituVR , I run strategy test and it worked perfectly .
I update again author. Please help to review it.

**Comment by acinmavi** [comment]
Hi @ChaituVR , could you please help to review again ? I update author and test again contract with 500 address . It's ok now.
And one question : If my pull request is accepted , does my strategy appear on https://snapshot.org and https://demo.snapshot.org ? I need this to test voting.
Best regards.

**Comment by ChaituVR** [comment]
@acinmavi Like i mentioned `contract-call` strategy should work fine in this case, no need of a new strategy, Can try that strategy?

**Comment by acinmavi** [comment]
Thanks @ChaituVR I could use contract-call for that strategy
Regards.

**Pull request closed without merging**